### PR TITLE
exercise-1

### PR DESCRIPTION
**Pull request by FrancoisLeclerc**
exercise-1

### CODE REVIEW

**Comment by nyanofthemoon** [comment]
@FrancoisLeclerc francois

File: index.html [hunk 1]
```diff
@@ -9,6 +9,29 @@
   </head>
   
   <body>
-    <!-- write your html here-->
+    
+    <div class= "nav-container">
```
Comment (nyanofthemoon) [inline index.html]: @FrancoisLeclerc Please use the HTML5 tag for navigation.

File: index.html [hunk 2]
```diff
@@ -9,6 +9,29 @@
...
+      <ul>
```
Comment (nyanofthemoon) [inline index.html]: @FrancoisLeclerc Please follow specification: "Fix the browser default padding and margin on your ul element by resetting it to 0"

**Comment by ziad-saab** [comment]
:+1:

**Pull request closed without merging**